src/main/java/com/internship/frejaeidjmeterplugin/jmeter/sampler/MobileClientSampler.java

```diff
@@ -0,0 +1,44 @@
+package com.internship.frejaeidjmeterplugin.jmeter.sampler;
+
+import com.internship.frejaeidjmeterplugin.jmeter.frejaRequests.MobileClientService;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import org.apache.jmeter.samplers.SampleResult;
+
+public class MobileClientSampler implements GenericSampler {
+    private final MobileClientService mobileClientService;
+    
+    public MobileClientSampler() throws Exception {
+        mobileClientService = new MobileClientService();
+    }
+    
+    @Override
+    public SampleResult sample(String entry) {
+        SampleResult sampleResult = new SampleResult();
+        sampleResult.sampleStart();
+        try {
+            mobileClientService.openSecureConnection();
+            setSampleResult(sampleResult, "openSecureConnection", true, "Freja eID Response: DELIVERED", "DELIVERED", "The open secure connection request was delivered");
```

Comment:
Maybe we should use SUCCESSFULLY instead of DELIVERED in context of openSecureConnection.